vllm/engine/llm_engine.py

```diff
@@ -346,6 +346,8 @@ def get_tokenizer_for_seq(sequence: Sequence) -> AnyTokenizer:
         # Create the scheduler.
         # NOTE: the cache_config here have been updated with the numbers of
         # GPU and CPU blocks, which are profiled in the distributed executor.
+        Scheduler = resolve_obj_by_qualname(
```

Comment:
if it is a string, use `resolve_obj_by_qualname` .

if not, it should be directly used as a class (and maybe assert it inherits from a base class?)